.gitignore

```diff
@@ -30,3 +30,5 @@ build/
 ### VS Code ###
 .vscode/
 /postgres-data/
+
+application.yml
```

Comment:
I don't see the logic behind it. We shouldn't do anything that isn't related to the task in the PR, and I really don't see the point in adding application.yml to the gitignore file.